modules/nf-core/purecn/run/main.nf

```diff
@@ -0,0 +1,100 @@
+process PURECN_RUN {
+    tag "$meta.id"
+    label 'process_medium'
+
+    // WARN: Version information not provided by tool on CLI. Please update version string below when bumping container versions.
+    conda "bioconda::bioconductor-purecn=2.4.0 bioconda::bioconductor-txdb.hsapiens.ucsc.hg38.knowngene=3.16.0 bioconductor-txdb.hsapiens.ucsc.hg19.knowngene=3.2.2 bioconda::bioconductor-org.hs.eg.db=3.16.0"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/mulled-v2-582ac26068889091d5e798347c637f8208d77a71:a29c64a63498b1ee8b192521fdf6ed3c65506994-0':
+        'quay.io/biocontainers/mulled-v2-582ac26068889091d5e798347c637f8208d77a71:a29c64a63498b1ee8b192521fdf6ed3c65506994-0' }"
+
+    input:
+    tuple val(meta), path(vcf)
+    path tumor_coverage
+    path normal_coverage
+    path normal_db
+    path intervals
+    path mapping_bias
+    path blacklist
+    val genome
```

Comment:
These, and the other `val` lines can be supplied as `$args`, no need to have them as parameters.
`mapping bias` is also optional if I recall, so the same thing applies, likewise for `blacklist`. This should simplify the inputs a lot.

Reply:
Changes done.